src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -1019,9 +1027,9 @@ private void HandleClusterEvent(ClusterEvent.IClusterDomainEvent e)
                 case ClusterEvent.MemberDowned md:
                     if (md.Member.UniqueAddress == Cluster.SelfUniqueAddress)
                     {
+                        Log.Info("{0}: Self downed, stopping ShardRegion [{1}]", TypeName, Self.Path);
                         Context.Stop(Self);
                     }
-                    Log.Info("Self downed, stopping ShardRegion [{0}]", Self.Path);
```

Comment:
This was always logging, whether it was the same node or not.

Reply:
Good catch